plugins/GSdx/GSSetting.cpp

```diff
@@ -57,9 +58,16 @@ const char* dialog_message(int ID, bool* updateText) {
 				"Works as a speedhack for: Steambot Chronicles.";
 		case IDC_SKIPDRAWHACK:
 		case IDC_SKIPDRAWHACKEDIT:
-			return "Skips drawing n surfaces completely. "
-				"Use it, for example, to try and get rid of bad post processing effects."
-				" Try values between 1 and 100.";
+			return "Completely skips drawing up to the surface specified here from the surface specified in the box to the left.\n\n"
+				"Use it, for example, to try and get rid of bad post processing effects.\n"
+				"Increase this value and keep the value in the box next to this set to \"1\" to keep the same skipdraw behaviour as before Skipdraw Range was introduced.\n"
+				"Try values between 1 and 100.";
+		case IDC_SKIPDRAWOFFSET:
+		case IDC_SKIPDRAWOFFSETEDIT:
+			return "Completely skips drawing surfaces from the surface specified here up to the surface specified in the box to the right.\n\n"
```

Comment:
Is there any way to use the same message for both boxes?

Reply:
Done.